builder/index.html

```diff
@@ -23,7 +23,12 @@
         <script src="js/ui-bootstrap-tpls-0.14.3.min.js?version=dev"></script>
         <script src="js/showdown-1.9.0.min.js?version=dev"></script>
         <script type="text/ng-template" id="deprecatedMethod.html">
-          <div>This generic method is <b>deprecated</b>.</div>
+          <div>
+            <div>This generic method is <b>deprecated</b>.</div>
+            <div class="deprecated-message" ng-if="method_call"><b>↳</b> {{method_call.deprecated}}</div>
```

Comment:
Maybe you can add a comment telling where it will be used in the interface

Reply:
Are the comment in the same order ? Maybe a comment line for just above the concerned one